checklists/alz_checklist.en.json

```diff
@@ -1220,127 +1183,298 @@
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Use IP Groups or IP prefixes to reduce number of IP table rules",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "0da83bb1-2f39-49af-b5c9-835fc455e3d1",
+            "id": "D07.12",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/ip-groups"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Avoid wildcards as a source IP for DNATS, such as * or any, you should specify source IPs for incoming DNATs",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "c44c6f0e-1642-4a61-a17b-0922f835c93a",
+            "id": "D07.13",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/tutorial-firewall-dnat"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Prevent SNAT Port exhaustion by monitoring SNAT port usage, evaluating NAT Gateway settings, and ensuring seamless failover. If the port count approaches the limit, it’s a sign that SNAT exhaustion might be imminent.",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "7371dc21-251a-47a3-af14-6e01b9da4757",
+            "id": "D07.14",
+            "severity": "Medium",
+            "link": "https://learn.microsoft.com/en-us/azure/nat-gateway/tutorial-hub-spoke-nat-firewall"
+        },
+         {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Enable TLS Inspection",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "346840b8-1064-496e-8396-4b1340172d52",
+            "id": "D07.15",
+            "severity": "High",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/premium-features#tls-inspection"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "Use web categories to allow or deny outbound access to specific topics.",
+            "waf": "Performance",
+            "service": "Firewall",
+            "guid": "39990a13-915c-45f9-a2d3-562d7d6c4b7c",
+            "id": "D07.16",
+            "severity": "Low",
+            "link": "https://learn.microsoft.com/en-us/azure/firewall/premium-features#web-categories"
+        },
+        {
+            "category": "Network Topology and Connectivity",
+            "subcategory": "Firewall",
+            "text": "As part of your TLS inspection, plan for receiving traffic from Azure App Gateways for inspection.",
```

Comment:
Are we able to combine this with D07.15?

Reply:
This point is supposed to address how placement with AppGW is a best practice for FW, whereas 07.15 is to address the benefits of TLS alone. How would you suggest combining the links?